### ISSUE TITLE
Issue on page /quickstart-hub-admin/tasks-config.html

### ISSUE THREAD

**Issue by nickreich** [Infectious-Disease-Modeling-Hubs/hubDocs#99]
In section 5.5.1 the text about how to designate a output_type as required or optional is not fully clear for the mean output type. I think that if the mean is required, the block should look like
```
"mean": {
                            "output_type_id": {
                                "required": ["NA"],
                                "optional": null
                            }, ...
```
where `["NA"]` is kind of like an indicator that that option has been selected. But the text needs to be clearer.

**Comment by nickreich** [comment]
Noting that there is a more complete description in the schema file, but it would be useful to have here as well.
https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.1/tasks-schema.json

**Comment by micokoch** [comment]
Addressed in #102 .
